src/pages/product-list-page/product-list-filter/category-filter/category-filter.js

```diff
@@ -23,9 +23,14 @@ const CategoryFilter = ({ selectedCategory }) => {
 
   const { categoryFilter } = filters;
 
-  const categories = [
-    ...new Set(filterData.map((product) => JSON.stringify(product.category)))
-  ].map(JSON.parse);
+  const categories = filterData
+    .map((product) => product.category)
+    .filter(
+      (category, index, self) =>
+        self
```

Comment:
What are you trying to do here, it is hard for me to understand

Reply:
I use it to create filter checkbox values

Reply:
fixed